files/ru/web/javascript/reference/statements/throw/index.md

```diff
@@ -37,13 +37,7 @@ throw true; // генерирует исключение, значением к
 Можно указать объект в качестве исключения. Затем можно получить ссылку на этот объект и доступ ко всем его свойствам в блоке catch. Следующий пример создаёт объект ошибки, который имеет тип UserException, и используется для генерации исключения.
 
 ```js
-function UserException(message) {
-  this.message = message;
-  this.name = "Исключение, определённое пользователем";
-}
-function getMonthName(mo) {
-  mo = mo - 1; // Нужно скорректировать номер месяца согласно индексам массива (1 = Jan, 12 = Dec)
-  const months = [
+const months = [
```

Comment:
I think it would be more correct to take a current example from the English version